environments/staging/aws-resources.yml

```diff
@@ -21,6 +21,7 @@ es7-staging.instance_id: i-087ff77521605d9b9
 es8-staging: 10.201.40.31
 es8-staging.instance_id: i-0d9e988943761e595
 formplayer-efs: fs-ba70cd39.efs.us-east-1.amazonaws.com
+efs-staging-mount-dns: shared-efs.staging.commcare.local
```

Comment:
Is this generated in terraform somewhere?

Reply:
Yes @dannyroberts, We had executed it through [PR](https://github.com/dimagi/commcare-cloud/pull/4560).

Reply:
Ah okay, great. Makes sense, thanks!